docs.md

```diff
@@ -488,7 +488,16 @@ renaming_overrides_prefixing = true
 "MyType" = "my_cool_type"
 "my_function" = "BetterFunctionName"
 
-# Table of things to add to the body of any struct, union, or enum that has the
+# Table of things to prepend to the body of any struct, union, or enum that has the
+# given name. This can be used to add things like methods which don't change ABI,
+# mark fields private, etc
+[export.body_prepend]
```

Comment:
Could we call this `pre_body`? I think it reads better but... your call, if you feel strongly about this feel free to leave it as-is.

Reply:
Will do.